digiwf-integrations/digiwf-s3-integration/digiwf-s3-integration-example/pom.xml

```
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>example-s3-integration</artifactId>
+    <name>example-s3-integration</name>
```

````suggestion
    <artifactId>digiwf-s3-integration-example</artifactId>
    <name>digiwf-s3-integration-example</name>
````

Rename the artifactId and name that they match the directories name.